### ISSUE TITLE
Project Abandoned?

### ISSUE THREAD

**Issue by mladinox** [f1nalspace/final_game_tech#159]
Good Idea, but is this project abandoned? not activity from 1 year ago,..
building with clang++ and g++, trows errors,..

**Comment by mladinox** [comment]
,,and not compiles,..

**Comment by f1nalspace** [comment]
Hi, thanks for the Feedback. Actually the project is still active and the "develop" branch is more advanced than the main branch.
So if you can double-check with the develop branch, that would be great.
Regarding compile errors. I haven't checked other compilers except MSVC for a while, so there may be some warnings threated as errors now or some missing includes.

**Comment by mladinox** [comment]
Hi, thanks for responding, error throws when compiling in --Debian 11--, gcc/g++ 10 and clang++ 11,..

**Comment by f1nalspace** [comment]
I tried compiling the "FPL_C" and "FPL_OpenGL" from the demos folder with GCC and clang and both compiled and run just fine.

```
$ gcc --version
gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0
```

```
$ clang --version
Ubuntu clang version 14.0.0-1ubuntu1.1
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin

```

But i had to fix the Makefile, i missed a rename of the main translation unit a while ago.
This may be required for other demos as well.

**Comment by f1nalspace** [comment]
Closed due to duplicate of #161
